apps/web/pages/api/user/avatar.ts

```diff
@@ -42,14 +42,30 @@ async function getIdentityData(req: NextApiRequest) {
     : null;
 
   if (username) {
-    const user = await prisma.user.findFirst({
+    const users = await prisma.user.findMany({
```

Comment:
Even though I have tried to use avatarUrl everywhere which doesn't use this endpoint. There are still places where this endpoint is used and so it's important to fix it as well.